clients/ctl/admin-ui/src/features/common/form/inputs.tsx

```diff
@@ -33,10 +33,11 @@ export const CustomTextInput = ({
   tooltip,
   disabled,
   ...props
-}: InputProps & FieldHookConfig<string>) => {
-  const [field, meta] = useField(props);
+}: InputProps & FieldHookConfig<string | undefined>) => {
```

Comment:
Nit: could use a code comment that we're doing this for all our inputs b/c of formik convenience.